telegram/_bot.py

```diff
@@ -4591,6 +4594,12 @@ async def get_updates(
         else:
             arg_read_timeout = self._request[0].read_timeout or 0
 
+        read_timeout = (
+            (arg_read_timeout + timeout.total_seconds())
+            if isinstance(timeout, dtm.timedelta)
+            else (arg_read_timeout + timeout if isinstance(timeout, int) else arg_read_timeout)
+        )
```

Comment:
```suggestion
        read_timeout = (
            (arg_read_timeout + timeout.total_seconds())
            if isinstance(timeout, dtm.timedelta)
            else (arg_read_timeout + timeout if timeout else arg_read_timeout)
        )
```
ah I don't think I've ever seen nested ternary if statements in python haha. Regarding the change, we should check for truthiness of `timeout` like before, because your case didn't cover if timeout was a float.